.github/workflows/tests.yml

```diff
@@ -12,21 +12,13 @@ jobs:
             fail-fast: true
             matrix:
                 python-version: ['3.8', '3.9', '3.10']
-                poetry-version: ['1.4.1']
-                os: [ubuntu-latest]
-        runs-on: ${{ matrix.os }}
```

Comment:
Taking the opportunity to simplify the action, keeping only the parameters which are needed in the matrix.